client/app/scripts/charts/edge.js

```diff
@@ -5,6 +5,20 @@ import classNames from 'classnames';
 import { enterEdge, leaveEdge } from '../actions/app-actions';
 import { encodeIdAttribute, decodeIdAttribute } from '../utils/dom-utils';
 
+
+function getAdjacencyClass(id) {
+  const topologyId = id.split('---');
+  const from = topologyId[0].split(';');
+  const to = topologyId[1].split(';');
+  if (from[1] !== undefined && to[1] !== undefined) {
+    from[1] = from[1].slice(1, -1);
+    to[1] = to[1].slice(1, -1);
+    if ((from[1] === 'persistent_volume' || from[1] === 'storage_class' || from[1] === 'persistent_volume_claim') || (to[1] === 'persistent_volume' || to[1] === 'storage_class' || to[1] === 'persistent_volume_claim')) {
```

Comment:
- This conditional is a bit long 
- assigning back into the array is confusing to read.
- Splitting into two fns might make it easier to parse!

```js
function isStorageNode(nodeId) {
  ...
}

function getAdjacencyClass(id) {
  const [ from, to ] = id.split('---');
  if (isStorageNode(from) || isStorageNode(to)) {
    ...
  }
  ...
}
```